Rollbar/RollbarConfiguration.m

```diff
@@ -70,16 +50,16 @@ - (id)init {
         self.scrubWhitelistFields = [NSMutableSet new];
         self.telemetryViewInputsToScrub = [NSMutableSet new];
 
-        self.notifierName = NOTIFIER_NAME;
-        self.notifierVersion = NOTIFIER_VERSION;
-        self.framework = FRAMEWORK;
-        self.captureIp = CaptureIpFull;
+        _notifierName = NOTIFIER_NAME;
```

Comment:
I assume it is better syntax for implicitly synthesized backing data fields?

Reply:
`self.notifierName = ...` is a message send, `_notifierName = ...` is a direct ivar access. It is signifies that no extra work is being done in some setter that might exist, instead that these are just plain properties.

Reply:
makes sense. thanks!